### PR TITLE
feat: arbitration approval UI

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/wizards/Sell/4_SelectArbitrator/components/ArbitratorSelectorForm.jsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ class ArbitratorSelectorForm extends Component {
       if (!user) {
         text = arbitratorAddr + ' - Loading...';
       } else {
-        text = `${index} - ${user.username || compactAddress(arbitratorAddr, 3)}${user.location ? ' from ' + user.location : ''} - ${user.upCount || 0}↑  ${user.downCount || 0}↓`;
+        text = `${index + 1} - ${user.username || compactAddress(arbitratorAddr, 3)}${user.location ? ' from ' + user.location : ''} - ${user.upCount || 0}↑  ${user.downCount || 0}↓`;
```
Comment (jrainville) [inline src/js/wizards/Sell/4_SelectArbitrator/components/ArbitratorSelectorForm.jsx]: I think we use this template 3 times? Might we worth to create a function or component

File: src/js/pages/SellerApproval/index.jsx [hunk 2]
```diff
@@ -37,9 +66,44 @@ class SellerApproval extends Component {
         <h2 className="mb-4">Seller Management</h2>
         <h3 className="mb-2">Accept all sellers</h3>
         {<BootstrapSwitchButton onlabel="ON" offlabel="OFF" checked={acceptsEveryone} onChange={this.onToggleCheckbox}/>}
-
-        <p><br /><br /><br />TODO: show list of requests for arbitrator</p>
         
+        { !acceptsEveryone && <Fragment>
```
[the remark this answers is not in the document]
Comment (richard-ramos) [reply src/js/pages/SellerApproval/index.jsx]: I'll add it to pivotal tracker!

File: src/js/features/arbitration/saga.js [hunk 3]
```diff
@@ -116,6 +117,30 @@ export function *onLoadArbitration() {
...
 
+export function *onGetArbitratorApprovalRequests() {
+  yield takeEvery(GET_ARBITRATION_REQUESTS, doGetArbitratorApprovalRequests);
+}
+
+export function *doGetArbitratorApprovalRequests({address}) {
+  try {
+    const events = yield ArbitrationLicense.getPastEvents('ArbitratorRequested', {fromBlock: 1, filter: {arbitrator: address} });
+    const requests = [];
+    for (let i = 0; i < events.length; i++) {
```
[the remark this answers is not in the document]
Comment (richard-ramos) [reply src/js/features/arbitration/saga.js]: I'm so confused by this image and the comment

[the remark this answers is not in the document]
Comment (jrainville) [reply src/js/features/arbitration/saga.js]: LOL oops, I posted the wrong link, I guess my copy command didn't work.
I meant to link this https://github.com/status-im/status-teller-network/blob/master/src/js/features/escrow/saga.js#L168
It's an example of how to use `all`

**Comment by jrainville** [comment]
Just need to fix the tests

**Pull request merged**